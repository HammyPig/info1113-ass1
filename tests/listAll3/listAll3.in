ADD MEMBER coolman
ADD MEMBER coolGUY
ADD COLLECTION books.csv
RENT 100000 111112
RENT 100000 111113
RENT 100001 111117
LIST ALL LONG