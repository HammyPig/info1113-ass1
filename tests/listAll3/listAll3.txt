recognises when books are being rented