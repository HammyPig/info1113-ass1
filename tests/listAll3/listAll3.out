user: Success.

user: Success.

user: 17 books successfully added.

user: Success.

user: Success.

user: Success.

user: 111111: To Kill a Mockingbird (Harper Lee, Historical Fiction)
Currently available.

111112: The Hobbit (J.R.R. Tolkien, Fantasy)
Rented by: 100000.

111113: The Iliad (Homer, Poetry)
Rented by: 100000.

111114: The Odyssey (Homer, Poetry)
Currently available.

111115: The Hitchhiker's Guide to the Galaxy (Douglas Adams, Comedy)
Currently available.

111116: So Long and Thanks for all the Fish (Douglas Adams, Comedy)
Currently available.

111117: Batman Volume 1 (Bob Kane, Comic)
Rented by: 100001.

111118: Harry Potter 1 (J.K. Rowling, Fantasy)
Currently available.

111119: Harry Potter 2 (J.K. Rowling, Fantasy)
Currently available.

111120: Harry Potter 3 (J.K. Rowling, Fantasy)
Currently available.

111121: Harry Potter 4 (J.K. Rowling, Fantasy)
Currently available.

111122: Harry Potter 5 (J.K. Rowling, Fantasy)
Currently available.

111123: Harry Potter 6 (J.K. Rowling, Fantasy)
Currently available.

111124: Wasteland (T.S. Eliot, Poetry)
Currently available.

111125: Biggles in the Battle (W.E. Johns, War)
Currently available.

111126: Aeneid (Virgil, Epic)
Currently available.

111127: Halloween (Jamie Lee Curtis, Horror)
Currently available.

user: Ending Library process.
